debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.1.100-8) unstable; urgency=medium
+  [ Dmitry Eremin ]
+  * Revert Mercury version to v2.0.1rc3
```

Comment:
Why this is relevant?

Reply:
I realize on my local machine there are no timeouts for RPCs with old version of Mercury. So, I'd like to see behavior on HW testing. Also, in 2.0 branch we mostly don't have this issue and it use old version of Mercury. So, I tent to think the root cause of the issue is a bug in Mercury which just looses some RPCs.